components/member-list-item/index.js

```diff
@@ -8,6 +8,11 @@ import { motion } from 'framer-motion';
 const PreviewMember = ({ memberDetails }) => {
   const { username } = memberDetails;
   const socialMedia = ['twitter_id', 'github_id', 'linkedin_id', 'instagram_id'];
+  const fullName = `${memberDetails.first_name + ' ' + memberDetails.last_name}`;
```

Comment:
NIT: why not destructure `first_name` and `last_name` as well at line no 9?

Reply:
will be done